include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -247,95 +230,164 @@ struct TypeInfo;
 struct Value;
 struct ModelMetadata;
 
+/** \brief The Env (Environment)
+*
+* The Env holds the logging state used by all other objects.
+* <b>Note:</b> One Env must be created before using any other Onnxruntime functionality
+*/
 struct Env : Base<OrtEnv> {
-  Env(std::nullptr_t) {}
+  explicit Env(std::nullptr_t) {} ///< \brief Create an empty Env object, must be assigned a valid one to be used 
+
+  /** \brief Creates an Env
+  *
+  * Wraps OrtApi::CreateEnv
+  *
+  * \param[in] logging_level
+  * \param[in] logid
```

Comment:
>logid [](http://example.com/codeflow?start=15&length=5)

woud these be worth documenting?

Reply:
It's mostly a framework for if we have something more worth saying per parameter. It's an error to leave one out if you list any.

Reply:
as a more specific question, what's a "logid"? :)

Reply:
Yeah, these will be filled in later by probably Scott.